source/common/local_reply/local_reply.cc

```diff
@@ -22,11 +22,7 @@ class BodyFormatter {
 
   BodyFormatter(const envoy::config::core::v3::SubstitutionFormatString& config)
       : formatter_(Formatter::SubstitutionFormatStringUtils::fromProtoConfig(config)),
-        content_type_(
-            config.format_case() ==
-                    envoy::config::core::v3::SubstitutionFormatString::FormatCase::kJsonFormat
-                ? Http::Headers::get().ContentTypeValues.Json
-                : Http::Headers::get().ContentTypeValues.Text) {}
+        content_type_(Formatter::SubstitutionFormatStringUtils::getContentType(config)) {}
```

Comment:
nit: may be you can inline this initialization 

!config.content_type().empty()? config.content_type() : config.format_case() ==
                 envoy::config::core::v3::SubstitutionFormatString::FormatCase::kJsonFormat
             ? Http::Headers::get().ContentTypeValues.Json
             : Http::Headers::get().ContentTypeValues.Text;
and avoid the method?